package.json

```diff
@@ -136,7 +136,7 @@
         "eslint-pre-push": "scripts/eslint-pre-push.sh",
         "flake8": "flake8 .",
         "flake8_files": "flake8",
-        "headless": "kbase-narrative --no-browser --NotebookApp.allow_origin='*' --ip=127.0.0.1 --port=32323",
+        "headless": "kbase-narrative --no-browser --ServerApp.allow_origin='*' --ip=127.0.0.1 --port=32323",
```

Comment:
Notebook 6.5.6 assigns certain configs to the jupyter-server app. It's unclear whether those are actually picked up or not, though, but this one's benign enough.